README.md

```diff
@@ -14,26 +14,7 @@ Subscribe to the following repo to be notified of announcements and updates abou
 
 ## REST API Reference
 
-The following reference pages are available to describe the Azure App Configuration API surface in detail.
-
-**Resources**
-  * [Keys](./docs/REST/keys.md)
-  * [Key-Values](./docs/REST/kv.md)
-  * [Labels](./docs/REST/labels.md)
-  * [Locks](./docs/REST/locks.md)
-  * [Revisions](./docs/REST/revisions.md)
-
-**Protocol**
-  * [Authentication](./docs/REST/authentication/index.md)
-  * [Authorization](./docs/REST/authorization/index.md)
-  * [Consistency Model](./docs/REST/consistency.md)
-  * [Common Headers](./docs/REST/headers.md)
-  * [Throttling](./docs/REST/throttling.md)
-  * [Versioning](./docs/REST/versioning.md)
-
-**Development**
-  * [Fiddler](./docs/Development/fiddler.md)
-  * [Postman](./docs/Development/postman.md)
+Access full functionalities of App Configuration via its [REST APIs]((https://docs.microsoft.com/azure/azure-app-configuration/rest-api)) from any programming languages including those that don't have SDKs or development tools you often use.
```

Comment:
"Access full functionalities" should be "Access the full functionality".

If I was a developer coming back to look up the reference, this sentence would be a bit much. It probably doesn't need to be anything more than "The Azure App Configuration REST API reference has moved [here](https://docs.microsoft.com/azure/azure-app-configuration/rest-api)."

Reply:
I was debating myself which style to go with, 'descriptive' vs. 'simple'. So 'simple' it is. Let me know if you like the new one. I didn't mention "move" because it's obvious for returning users and it's meaningless for new users. I optimize for new users :).

Reply:
Looks great.